adapter/stdioLogger/config/config.proto

```diff
@@ -50,30 +45,24 @@ message Params {
         STRUCTURED = 1;
     }
 
-    // The name of the attribute to treat as the payload of the logs message.
-    // If payload_attribute is not specified, there will be no payload specified
-    // in the log entry written. All provided attributes will be written out
-    // as logging metadata.
-    string payload_attribute = 3;
-
     // Format of the value of the payload attribute.
-    PayloadFormat payload_format = 4;
+    PayloadFormat payload_format = 2;
 
     // The name of the attribute that will be used to derive log severity.
     // If this attribute is not found in the set of attributes passed to this
     // implementation, a default severity of INFO will be used.
-    string severity_attribute = 5;
+    string severity_attribute = 3;
 
     // The name of the attribute that will be used to derive the timestamp
     // for the corresponding log enty. If this attribute is not specified, the
     // value from time.Now() will be used as the timestamp for the log.
-    string timestamp_attribute = 6;
+    string timestamp_attribute = 4;
```

Comment:
Perhaps. I've moved it to aspect config for now. OK with leaving it in for this CL with a subsequent move? We can refactor after this CL (as it involves multiple CLs across repos).